scripts/deploy.sh

```diff
@@ -56,12 +56,15 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   export DEPLOY_HAPROXY="yes"
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
+    # remove unused variables file
+    rm /etc/openstack_deploy/user_variables.yml
```

Comment:
That file is laid down no mater what in the aio, I can still remove the original.  So this isn't actually a problem.

```
- name: Set the user_variables
  config_template:
    src: user_variables.aio.yml.j2
    dest: /etc/openstack_deploy/user_variables.yml
    config_overrides: "{{ user_variables_overrides | default({}) }}"
    config_type: yaml
```